sonar-plugin-api/src/main/java/org/sonar/api/test/MutableTestPlan.java

```diff
@@ -19,6 +19,10 @@
  */
 package org.sonar.api.test;
 
+/**
+ * @deprecated since 5.6. Feature will be removed without any alternatives.
+ */
+@Deprecated
 public interface MutableTestPlan extends TestPlan<MutableTestCase> {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)